### PR TITLE
Peer review of OSD docs

### PR DESCRIPTION
**Pull request by bergerhoffer**
Preview: https://deploy-preview-34101--osdocs.netlify.app/openshift-dedicated/latest/welcome/index.html

### CODE REVIEW

File: administering_a_cluster/dedicated-admin-role.adoc [hunk 1]
```diff
@@ -1,7 +1,8 @@
 [id="dedicated-administrator-role"]
 = The dedicated-admin role
 include::modules/common-attributes.adoc[]
-:context: dedicated-administrator
+:context: dedicated-administrator-role
```
Comment (vikram-redhat) [inline administering_a_cluster/dedicated-admin-role.adoc:4]: dedicated-admin-role?

File: modules/policy-shared-responsibility.adoc [hunk 2]
```diff
@@ -9,8 +9,9 @@
 
 The customer and Red Hat share responsibility for the monitoring and maintenance of an {product-title} cluster. This documentation illustrates the delineation of responsibilities by area and task.
 
+[id="policy-shared-responsibility-incident-operations-management_{context}"]
```
Comment (vikram-redhat) [inline modules/policy-shared-responsibility.adoc:12]: Should this be `incident-operations-management_{context}`. Same for the other id in this file.

File: administering_a_cluster/dedicated-admin-role.adoc (hunk 1 above)
Comment (bergerhoffer) [reply administering_a_cluster/dedicated-admin-role.adoc:4]: Good catch, I missed that the file name was different too. Will update

File: modules/policy-shared-responsibility.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/policy-shared-responsibility.adoc:12]: @vikram-redhat I had to add in a lot of missing IDs for the assemblies, and some of these titles alone could potentially conflict. To be safe, I started all of them with the main assembly's ID + the heading. Unless I'm missing something, it's not really that important what we call these IDs (unless you object to the length of the name).

File: administering_a_cluster/dedicated-admin-role.adoc (hunk 1 above)
Comment (sfortner-RH) [reply administering_a_cluster/dedicated-admin-role.adoc]: Yes, there is a dedicated-admin and cluster-admin roles in OSD/ROSA/ARO.

File: modules/upgrade-cli.adoc [hunk 3]
```diff
@@ -8,39 +8,42 @@
 = Upgrading your cluster through the CLI
 
 
-
 You can use the OpenShift Cluster Manager (OCM) CLI upgrade wizard to schedule automatic and manual upgrade policies, list the upgrade policy, and delete the upgrade policy for your {product-title} cluster.
 
+// TODO: Should there be a prereq here to have the ocm CLI installed? This is the first time I've come across it
```
Comment (sfortner-RH) [inline modules/upgrade-cli.adoc:13]: Honestly, engineering asked for this topic....but most customers use the UI for this or set up their upgrade policies at cluster creation. There is an OCM CLI guide being developed by the team handling the OCM docs.

File: modules/upgrade-cli.adoc [hunk 4]
```diff
@@ -8,39 +8,42 @@
...
+
+// TODO: Suggest another look at this procedure. Some of it looks like it might be sequential (like kicking off with the "ocm" command, but then next it'd be  choice of manual or automatic. And then it just gives some details about listing and deleting, which aren't necessarily a required part of the procedure. Also a question on the bold - does the command actually pop up a GUI wizard, or just walk you through the steps in the terminal?
```
Comment (sfortner-RH) [inline modules/upgrade-cli.adoc:15]: Just in the terminal.

File: monitoring/osd-configuring-the-monitoring-stack.adoc [hunk 5]
```diff
@@ -26,6 +26,8 @@ include::modules/osd-monitoring-moving-monitoring-components-to-different-nodes.
 * link:https://docs.openshift.com/container-platform/4.7/nodes/scheduling/nodes-scheduler-node-selectors.html[Placing pods on specific nodes using node selectors]
 * See the link:https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector[Kubernetes documentation] for details on the `nodeSelector` constraint
 
+// TODO: Same question about whether links out to OCP docs should explicitly mention that? (I won't point out any more, but could easily search to find other instances in the repo).
```
Comment (sfortner-RH) [inline monitoring/osd-configuring-the-monitoring-stack.adoc:29]: The end goal is to pull in the related OCP content and we're actively working with PM to figure out what pieces get pulled in. JQ wrote this before we had that capability, hence the linking out to OCP.

File: monitoring/osd-managing-metrics.adoc [hunk 6]
```diff
@@ -34,5 +39,7 @@ include::modules/osd-monitoring-exploring-the-visualized-metrics.adoc[leveloffse
 * xref:../monitoring/osd-troubleshooting-monitoring-issues.adoc#osd-troubleshooting-monitoring-issues[Troubleshooting monitoring issues]
 
 [id="managing-metrics-next-steps"]
-.Next steps
-xref:../monitoring/osd-managing-alerts.adoc#osd-managing-alerts[Alerts]
+== Next steps
+* xref:../monitoring/osd-managing-alerts.adoc#osd-managing-alerts[Alerts]
+
+// TODO: Why is alerts a next step if alerts aren't supported? Can this be removed?
```
Comment (sfortner-RH) [inline monitoring/osd-managing-metrics.adoc:45]: I'll check with JQ on this. She wrote this topic and I wasn't part of the review.

**Review by sfortner-RH — APPROVED** [review]
LGTM! We can tackle the TODO's in another PR. I answered any questions.

File: modules/policy-shared-responsibility.adoc (hunk 2 above)
Comment (vikram-redhat) [reply modules/policy-shared-responsibility.adoc:12]: I was asking about the anchor ids, and not the actual content in these docs. I am sure @bergerhoffer added those ids :) 

I believe @bergerhoffer is reducing them to remove the file name.

File: modules/upgrade-cli.adoc (hunk 4 above)
Comment (bergerhoffer) [reply modules/upgrade-cli.adoc:15]: Okay then yeah I would probably avoid the use of bold in the procedure then.

File: modules/policy-shared-responsibility.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/policy-shared-responsibility.adoc]: @vikram-redhat I went through and tried to reduce the size of the long ones I came across. I'm not sure why I forgot about the context making the IDs unique... I must have been delirious at that point :). If you see any other long ones that I missed, let me know and I'll update.

**Pull request merged**